src/test/java/com/google/cloud/pontem/AvroUtilTest.java

```diff
@@ -169,4 +188,9 @@ public void testGetAvroTypeFromSpannerType_basic() {
             .type(SchemaBuilder.builder().stringType()),
         AvroUtil.getAvroTypeFromSpannerType(Type.array(Type.timestamp()), true));
   }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void testIllegalType() {
```

Comment:
Similar case here could we make the reason for this being an illegal argument more explicit?